terraform/application/application.tf

```diff
@@ -49,6 +49,7 @@ module "web_application" {
 
   kubernetes_config_map_name = module.application_configuration.kubernetes_config_map_name
   kubernetes_secret_name     = module.application_configuration.kubernetes_secret_name
+  replicas                   = var.app_replicas
```

Comment:
I'd suggest this belongs with the `docker_image` parameter rather than with the Kubernetes ones.